extension/modules/log4moz.js

```diff
@@ -198,7 +198,9 @@ Logger.prototype = {
   },
 
   _parent: null,
-  get parent() this._parent,
+  get parent() {
+    this._parent;
+  },
```

Comment:
Because [name getter](https://github.com/derinb/personas-plus/blob/d4ea923097676224431d07207576b970f6395e15/extension/modules/log4moz.js#L183) is also coded as "return", I done it with same styling for formatting. Also I remember [older Firefox versions](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Firefox-specific_notes) do not like arrow functions so I better used conventional formatting. SeaMonkey also may benefit this.

If you want it to be changed let me know please.